### DISCUSSION TITLE
How to change size of added component?

### DISCUSSION THREAD

**Discussion by Lisiczny** [hneemann/Digital#902] [Q&A] [answered]
Like in a question, how to make added component bigger or smaller?
![3](https://user-images.githubusercontent.com/98484831/151224519-c92bb2f2-4df2-43f9-8c86-ba36d93cc831.PNG)

**Comment by hneemann** [comment, marked as answer]
Set the width and height properties in the circuit specific settings of the sub circuit.

**Reply by Lisiczny** [reply]
Really thanks for your answer! Didn't notice, that I have to make changes in the subcurcuit.
